src/main/java/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -1653,12 +1653,12 @@ else if(claim.allowGrantPermission(player) != null)
                         GriefPrevention.sendMessage(player, TextMode.Err, Messages.ManagersDontUntrustManagers, claim.getOwnerName());
                         return true;
 					}
-		    else if(claim.allowBuild(otherPlayer) == null && claim.allowBuild(player) != null)
+		    else if(claim.allowBuild(idToDrop) == null && claim.allowBuild(player) != null)
```

Comment:
allowbuild() requires Player and Material

Reply:
otherPlayer did not work because it is an offilnePlayer. how do i get a player object form an id?